.github/workflows/test_pull_requests.yml

```diff
@@ -27,142 +23,157 @@ jobs:
           macos_dev_target: 13.0
           arch: arm64
           python-arch: arm64
-          generator: "Unix Makefiles"
-          nproc: 3
           run_regressions: true
           pretty: "Mac arm64"
-#        - os: ubuntu-24.04
-#          arch: x86_64
-#          python-arch: x64
-#          generator: "Unix Makefiles"
-#          nproc: 4
-#          run_regressions: false
-#          pretty: "Ubuntu 24.04"
-#        - os: windows-2022
-#          arch: x86_64
-#          python-arch: x64
-#          generator: "Visual Studio 17 2022"
-#          nproc: 4
-#          run_regressions: false
-#          pretty: "Windows x64"
+        - os: ubuntu-24.04
+          arch: x86_64
+          python-arch: x64
+          run_regressions: true
+          pretty: "Ubuntu 24.04"
+        - os: windows-2022
+          arch: x86_64
+          python-arch: x64
+          run_regressions: true
+          pretty: "Windows x64"
```

Comment:
It's really exciting to see the platforms added back in now that CI will be speedy.